browser/services/platform.js

```diff
@@ -133,6 +133,34 @@ class Platform {
     });
   }
 
+  static getFreeDiskSpace(location) {
+    return Platform.identify({
+      win32: ()=> {
+        let disk = path.parse(location).root.charAt(0);
+        return pify(child_process.exec)(`powershell -command "& {(Get-WMIObject Win32_Logicaldisk -filter \"deviceid=\`'${disk}:\`'\") }"`).then((stdout) => {
+          return Promise.resolve(Number.parseInt(stdout));
+        }).catch(()=>{
+          return Promise.resolve();
+        });
+      },
+      darwin: ()=> {
+        return pify(child_process.exec)(`df -k ${location}`).then((stdout) => {
+          var lines = stdout.split('\n');
```

Comment:
I don't think we need cycle here, just 
``` JavaScript
let split = lines[1].replace( / +/g, ' ' ).split(' ');
// split[0] -> disk
// split[1] - > total 1k blocks
// split[2] - > total 1k blocks used
// split[3] - > total 1k blocks available
return Promise.resolve(Number.parseInt(split[3]));
```
we just need return the same units fro macOS and Windows.